dali/operators/random/rng_base_cpu.h

```diff
@@ -54,8 +54,10 @@ struct DistGen<true> {
   template <typename T, typename Dist, typename RNG>
   inline void gen(span<T> out, span<const T> in, Dist& dist, RNG &rng) const {
     assert(out.size() == in.size());
-    for (int64_t k = 0; k < out.size(); k++)
-      out[k] = ConvertSat<T>(dist(in[k], rng));
+    for (int64_t k = 0; k < out.size(); k++) {
+      auto n = dist.Generate(in[k], rng);
+      dist.Apply(out[k], in[k], n);
```

Comment:
Where `Apply` is implemented? I don't see it anywhere.

Reply:
It is implemented in those generators that have NoiseGen=true only.